docs/getting-started/detections-req.asciidoc

```diff
@@ -50,57 +52,79 @@ NOTE: For instructions about using Machine Learning jobs and rules, refer to <<m
 |Enable the Detections feature in your Kibana space
 |The `manage` privilege
 
-a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices where `<kib-space>` is the {kib} space name:
+a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices, where `<space-id>` is the {kib} space name:
+
+* `.alerts-security.alerts-<space-id>`
+* `.siem-signals-<space-id>`
+* `.lists-<space-id>`
+* `.items-<space-id>`
 
-* `.siem-signals-<kib-space>`
-* `.lists-<kib-space>`
-* `.items-<kib-space>`
-|{kib} space `All` privileges for the `Security` feature (see
+*NOTE*: If you are upgrading to {stack} 8.0.0 and later, users should have privileges for the `.alerts-security.alerts-<space-id>` AND `.siem-signals-<space-id>` indices. If you are newly installing the {stack}, then users do not need privileges for the `.siem-signals-<space-id>` index.
```

Comment:
```suggestion
*NOTE*: If you are upgrading to {stack} 8.0.0 or later, users should have privileges for the `.alerts-security.alerts-<space-id>` AND `.siem-signals-<space-id>` indices. If you are newly installing the {stack}, then users do not need privileges for the `.siem-signals-<space-id>` index.
```
"8.0.0 and later" appears a few places, but I think it should be "8.0.0 or later" because you can't upgrade to more than one version (i.e. 8.0.0 and 8.0.1) — you can only upgrade to one version at a time such as 8.0.0, or 8.0.1.